hydra-main/src/main/java/com/addthis/hydra/query/aggregate/QueryTask.java

```diff
@@ -122,6 +85,57 @@ public void run() {
         }
     }
 
+    private int readFrame(List<QueryTaskSource> taskSources, int maxReads) throws Exception {
```

Comment:
I assumed it's a "frame" because of AggregateConfig.FRAME_READER_READS. I'm fine with rename it to whatever.